sale_quotation_number/tests/test_sale_order.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import UserError
+from odoo.tests.common import TransactionCase
+
+
+class TestSaleOrder(TransactionCase):
+    def setUp(self, *args, **kwargs):
+        super(TestSaleOrder, self).setUp()
+        self.sale_order_model = self.env["sale.order"]
```

Comment:
```suggestion
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
        cls.sale_order_model = cls.env["sale.order"]
```

For test performances

Reply:
Hello @rousseldenis, I've done the changes as per your feedback, Kindly check it.